packages/dds/tree/src/test/shared-tree/sharedTree.spec.ts

```diff
@@ -221,83 +215,6 @@ describe("SharedTree", () => {
 		});
 	});
 
-	describe("requireSchema", () => {
```

Comment:
these tests are for a function, requireSchema, that is no longer in the file this file is testing. schematizingTreeView.spec.ts
 is the proper place for these, and it has relatively good coverage already, so only one of these two tests was ported to it.